Modules/MFT/include/MFT/BasicMFTDigitQcTask.h

```diff
@@ -0,0 +1,57 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   BasicMFTDigitQcTask.h
+/// \author Barthelemy von Haller
+/// \author Piotr Konopka
+///
+
+#ifndef QC_MODULE_MFT_MFTBASICMFTDIGITQCTASK_H
+#define QC_MODULE_MFT_MFTBASICMFTDIGITQCTASK_H
+
+#include "TH1F.h"
+#include "QualityControl/TaskInterface.h"
+
+class TH1F;
+
+using namespace o2::quality_control::core;
+
+namespace o2::quality_control_modules::mft
+{
+
+/// \brief Example Quality Control DPL Task
+/// It is final because there is no reason to derive from it. Just remove it if needed.
+/// \author Barthelemy von Haller
+/// \author Piotr Konopka
+class BasicMFTDigitQcTask /*final*/ : public TaskInterface // todo add back the "final" when doxygen is fixed
+{
+ public:
+  /// \brief Constructor
+  BasicMFTDigitQcTask() = default;
+  /// Destructor
+  ~BasicMFTDigitQcTask() override;
+
+  // Definition of the methods for the template method pattern
+  void initialize(o2::framework::InitContext& ctx) override;
+  void startOfActivity(Activity& activity) override;
+  void startOfCycle() override;
+  void monitorData(o2::framework::ProcessingContext& ctx) override;
+  void endOfCycle() override;
+  void endOfActivity(Activity& activity) override;
+  void reset() override;
+
+ private:
+  std::unique_ptr<TH1F> MFT_chip_index_H = nullptr;
```

Comment:
For class member variables, please use the camel case convention with `m` at the beginning, as specified in the coding guidelines https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names

Reply:
Done, MFT_chip_index_H replaced by mMFT_chip_index_H.